core/src/main/java/io/questdb/cutlass/http/processors/LineHttpProcessorState.java

```diff
@@ -0,0 +1,446 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2023 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.cutlass.http.processors;
+
+import io.questdb.cairo.CairoEngine;
+import io.questdb.cairo.CairoException;
+import io.questdb.cairo.CommitFailedException;
+import io.questdb.cairo.SecurityContext;
+import io.questdb.cairo.security.AllowAllSecurityContext;
+import io.questdb.cutlass.http.ConnectionAware;
+import io.questdb.cutlass.http.HttpServerConfiguration;
+import io.questdb.cutlass.line.tcp.*;
+import io.questdb.log.Log;
+import io.questdb.log.LogFactory;
+import io.questdb.log.LogRecord;
+import io.questdb.std.*;
+import io.questdb.std.str.StringSink;
+import io.questdb.std.str.Utf8Sink;
+
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class LineHttpProcessorState implements QuietCloseable, ConnectionAware {
+    private static final AtomicLong ERROR_COUNT = new AtomicLong();
+    private static final String ERROR_ID = generateErrorId();
```

Comment:
this is my ignorance, but what's the meaning of ERROR_ID? I guess it's a protocol thing?

Reply:
Not exactly. Influx V3 sends a request-id header I guess to trace errors. A similar idea here, we send back a unique error identifier and log more information in the log file with this unique ID.